upgrading/automated_upgrades.adoc

```diff
@@ -916,6 +933,44 @@ logging deployment.
 
 If the xref:upgrading-efk-logging-stack-script[script above] indicates your indices do not contain fields with a dot in the name, use the following steps to upgrade.
 
+. Review how to
+xref:../install_config/aggregate_logging.adoc#aggregate-logging-ansible-variables[specify logging Ansible variables]
+and update your Ansible inventory file to at least set the
+following required variable in the `[OSEv3:vars]` section:
++
+----
+[OSEv3:vars]
+
+openshift_logging_install_logging=true <1>
+----
+<1> Enables the ability to upgrade the logging stack.
+
+. Update any other `openshift_logging_*` variables that you want to override the
+default values for, as described in
+xref:../install_config/aggregate_logging.adoc#aggregate-logging-ansible-variables[Specifying Logging Ansible Variables].
++
+You can set the `openshift_logging_elasticsearch_replace_configmap` parameter to `true` to replace your
+`logging-elasticsearch` ConfigMap  with the current default values. In some cases, using an older
+ConfigMap can cause the upgrade to fail. The default is set to `false`. For more information, see the parameter
+in xref:../install_config/aggregate_logging.adoc#aggregate-logging-ansible-variables[specify logging Ansible variables].
```

Comment:
I copied these two steps from the following procedure. Not sure why I didn't put them here originally. Review as desired.